site/default/static/map.js

```diff
@@ -0,0 +1,34 @@
+module.exports = {
```

Comment:
This could possibly break in the future if, for example, can-compute adds a new file to their project. I think what we should do here is create this module on the fly.  Crawl through the `can-` packages and create this same configuration.